README.md

```diff
@@ -21,16 +25,22 @@ Dependencies
 The following dependencies are required to run Procrustes properly,
 
 * Python >= 2.7, or Python >= 3.6: http://www.python.org/
-* PIP >= 9.0: https://pip.pypa.io/
+* PIP >= 19.0: https://pip.pypa.io/
 * SciPy >= 1.0.0: http://www.scipy.org/
 * NumPy >= 1.14: http://www.numpy.org/
-* Nosetests >= 1.3.7: http://readthedocs.org/docs/nose/en/latest/
+* PyTest >= 5.3.0: https://docs.pytest.org/
+* PyTest-Cov >= 2.8.0: https://pypi.org/project/pytest-cov/
 
 
 Installation
 ------------
 
-To install using package manager, run:
+Download the latest version of Procrustes with the following command.
+```bash
+    git clone git@github.com:theochem/chemtools.git
```

Comment:
Change `chemtools` to `procrustes`.

Reply:
Fixed typo in d0f9730. Thank you for reminding this.